comp/core/status/render_helpers.go

```diff
@@ -181,6 +182,15 @@ func formatUnixTime(unixTime any) string {
 	}
 }
 
+// formatJSON formats the given value as JSON. The indent parameter is used to indent the entire JSON output.
+func formatJSON(value interface{}, indent int) string {
+	b, err := json.MarshalIndent(value, "", "  ")
+	if err != nil {
+		return fmt.Sprintf("Error formatting JSON: %s", err)
+	}
+	return strings.Join(strings.Split(string(b), "\n"), "\n"+strings.Repeat(" ", indent))
```

Comment:
Instead of having to do this split and re-join, you can just calculate this value ahead of time and pass it to MarshalIndent:

```
    var pad = strings.Repeat(" ", indent)
    b, err := json.MarshalIndent(value, pad, pad + "  ")
```

Reply:
Actually the following is what we need

```go
    var pad = strings.Repeat(" ", indent)
    b, err := json.MarshalIndent(value, pad,  "  ")
```

From the documentation of the function

> // MarshalIndent is like Marshal but applies Indent to format the output.
// Each JSON element in the output will begin on a new line beginning with prefix
// followed by one or more copies of indent according to the indentation nesting.